wrappers/wtypes.go

```diff
@@ -16,6 +16,15 @@
 
 package wrappers
 
+type LPARAM uintptr
+
+type RECT struct {
```

Comment:
This should be defined in windef.h.  While it is also included in wtypes.h, wtypes.h is automatically generated from wtypes.idl, where `RECT` is included for use by MIDL.  You'll notice that the `RECT` in wtypes.h is surrounded by `#ifndef _WINDEF_`, which will prevent it from ever being included on the C/C++ side.